### PR TITLE
Merge develop to master for 2.2.0-beta3 release.

### PR DESCRIPTION
**Pull request by dnduffy**
Merge develop to master for 2.2.0-beta3 release.

### CODE REVIEW

File: Test/Web/FunctionalTests/TestApps/Wa40Aspx/Wa40Aspx.csproj [hunk 1]
```diff
@@ -50,11 +50,11 @@
   <ItemGroup>
     <Reference Include="Microsoft.AI.ServerTelemetryChannel, Version=2.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\..\..\..\packages\Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.2.2.0-beta2\lib\net40\Microsoft.AI.ServerTelemetryChannel.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\packages\Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.2.2.0-beta3-build34090\lib\net40\Microsoft.AI.ServerTelemetryChannel.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.ApplicationInsights, Version=2.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\..\..\..\packages\Microsoft.ApplicationInsights.2.2.0-beta2\lib\net40\Microsoft.ApplicationInsights.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\packages\Microsoft.ApplicationInsights.2.2.0-beta3-build34090\lib\net40\Microsoft.ApplicationInsights.dll</HintPath>
```
Comment (SergeyKanzhelev) [inline Test/Web/FunctionalTests/TestApps/Wa40Aspx/Wa40Aspx.csproj]: Should we take a reference to the stable one?

Comment (dnduffy) [inline Test/Web/FunctionalTests/TestApps/Wa40Aspx/Wa40Aspx.csproj]: Yes, I thought I fixed all of them...

Comment (dnduffy) [inline Test/Web/FunctionalTests/TestApps/Wa40Aspx/Wa40Aspx.csproj]: They are fixed now.

File: Test/Web/FunctionalTests/TestApps/WebAppFW45/packages.config [hunk 2]
```diff
@@ -1,11 +1,11 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.ApplicationInsights" version="2.2.0-beta2" targetFramework="net451" />
-  <package id="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" version="2.2.0-beta2" targetFramework="net451" />
+  <package id="Microsoft.ApplicationInsights" version="2.2.0-beta3-build34090" targetFramework="net451" />
```
Comment (SergeyKanzhelev) [inline Test/Web/FunctionalTests/TestApps/WebAppFW45/packages.config]: should be fixed to `-beta3` without postfix

File: upgradeVersion.ps1 [hunk 3]
```diff
@@ -3,6 +3,8 @@
 
 $newVersion = .\NuGet.exe list "Microsoft.ApplicationInsights" -Source https://www.myget.org/F/applicationinsights -Pre -NonInteractive | Select-String -Pattern "Microsoft.ApplicationInsights " | %{$_.Line.Split(" ")} | Select -skip 1
 
+$newVersion ="2.2.0-beta3-build34090"
```
Comment (SergeyKanzhelev) [inline upgradeVersion.ps1]: Remove this line please

**Pull request merged**